### PR TITLE
⬛ Completed Card

### PR DESCRIPTION
**Pull request by Squidnugget77**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -121,6 +121,33 @@ <h2>A project for first-time contributions</h2>
         <!-- Keep one line of space above and below your card -->
         <!-- ========= Paste YOUR CARD directly BELOW this line ========= -->
 
+        <div class="card">
+			<p class="name">Luca Giannotti</p>
+			<p class="contact">
+			  <i class="fab fa-instagram"></i>
+			  <a href="https://www.twitter.com/your_user_handle" target="_blank">Your handle</a>
```
Comment (theAravinthM) [inline index.html:128]: Since you have not provided your Twitter profile, you can remove these lines.

**Review by theAravinthM — CHANGES_REQUESTED** [review]
Hey @Squidnugget77 
You need a little modification in order to get your PR merged. Kindly update and commit back, I have provided you with the comments to help you out.

File: index.html (hunk 1 above)
Comment (Squidnugget77) [reply index.html]: Okay, I removed it and sent another commit!

**Comment by theAravinthM** [comment]
Congratulations @Squidnugget77 on your PR Merge, and Thanks for your Contribution! Your card will be live on our Website in a moment!

Now you can follow the Next Steps. You are also welcome to join our Discord Server: https://discord.gg/tWkvS4ueVF and be a part of our community.

Happy contributing!

**Pull request merged**